docs/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rancher-on-aks.md

```diff
@@ -88,10 +88,12 @@ helm upgrade --install \
   ingress-nginx ingress-nginx/ingress-nginx \
   --namespace ingress-nginx \
   --set controller.service.type=LoadBalancer \
-  --version 4.0.18 \
+  --version 4.6.0 \
   --create-namespace
 ```
 
+Consult the [official ingress-nginx Helm chart changelog](https://github.com/kubernetes/ingress-nginx/blob/helm-chart-4.6.0/charts/ingress-nginx/CHANGELOG.md) to find the  Helm chart `--version` that supports your version of Kubernetes.
```

Comment:
From a docs maintenance perspective, it would be better if we used NGINX's `kubernetes-ingress` instead of the community-supported `nginx-ingress`, since the NGINX team does a better job at describing compatibility issues (https://docs.nginx.com/nginx-ingress-controller/technical-specifications/#supported-kubernetes-versions).

Reply:
...unfortunately, RKE and RKE2 ship with the Kubernetes-maintained project, and Rancher also contains annotations assuming that this chart is in use.

While users can look up the `appVersion` of a chart in advance of installation, then look at https://github.com/kubernetes/ingress-nginx/tree/main#supported-versions-table to see if it's compatible, the closest thing to a one-step page matching helm charts to Kubernetes versions is the CHANGELOG.